randovania/games/dread/json_data/Cataris.txt

```diff
@@ -1055,7 +1055,8 @@ Extra - asset_id: collision_camera_015
                   Heat/Cold Runs (Intermediate) and Heat Damage ≥ 200
               Any of the following:
                   Simple IBJ
-                  Morph Ball and Single-wall Wall Jump (Intermediate)
+                  # SWJ: https://youtu.be/eS4UULex4Vk
+                  Morph Ball and Single-wall Wall Jump (Intermediate) and Slide Jump (Beginner)
```

Comment:
You should also put in `Slide` as a required resource in cases like these.

Reply:
Ah, I hadn't considered the case that the player wouldn't have slide. Added.

Reply:
Well, you currently can't not have slide, because rdv doesn't let you not start with it, but the logic still accounts for it.